upoi/poi/models/poi.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+# from django.contrib.gis.db import models
+
+
+class Poi(models.Model):
+    name = models.CharField(max_length=100)
+    address = models.CharField(max_length=200)
+    phone = models.CharField(max_length=20)
+    email = models.CharField(max_length=50)
+    website = models.CharField(max_length=150)
+    # coordinates = models.PointField()
+    coordinates = models.CharField(max_length=100)
```

Comment:
You can add a TODO comment here 

```suggestion
    # TODO: use PointField to save coordinates
    # coordinates = models.PointField()
    coordinates = models.CharField(max_length=100)
```